lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/postgres/AbstractPostgresKgStore.scala

```diff
@@ -6,28 +6,31 @@ import slick.jdbc.PostgresProfile
 
 abstract class AbstractPostgresKgStore(protected val dbConfigProvider: DatabaseConfigProvider) extends HasDatabaseConfigProvider[PostgresProfile] {
   import profile.api._
-  
+
   lazy val kgEdges = TableQuery[KgEdgeTable]
   lazy val kgEdgeLabels = TableQuery[KgEdgeLabelTable]
   lazy val kgEdgeKgSources = TableQuery[KgEdgeKgSourceTable]
   lazy val kgNodes = TableQuery[KgNodeTable]
   lazy val kgNodeLabels = TableQuery[KgNodeLabelTable]
+  lazy val kgNodeLabelEdges = TableQuery[KgNodeLabelEdgeTable]
+  lazy val kgNodeLabelEdgeKgSources = TableQuery[KgNodeLabelEdgeKgSourceTable]
+  lazy val kgNodeLabelKgSource = TableQuery[KgNodeLabelKgSourceTable]
   lazy val kgNodeKgSources = TableQuery[KgNodeKgSourceTable]
   lazy val kgSources = TableQuery[KgSourceTable]
 
   private class KgEdgeTable(tag: Tag) extends Table[(String, String, String, String, String)](tag, "kg_edge") {
     def id = column[String]("id", O.PrimaryKey)
-    def objectNodeId = column[String]("object_node_id")
+    def objectKgNodeId = column[String]("object_kg_node_id")
     def predicate = column[String]("predicate")
     def sentences = column[String]("sentences")
-    def subjectNodeId = column[String]("subject_node_id")
+    def subjectKgNodeId = column[String]("subject_node_id")
```

Comment:
You need some NOT NULL constraints, too? Not sure how Slick handles this.

Reply:
I found this in the docs http://scala-slick.org/doc/3.0.0/schemas.html
> NotNull, Nullable
Explicitly mark the column as nullable or non-nullable when creating the DDL statements for the table. Nullability is otherwise determined from the type (Option or non-Option). There is usually no reason to specify these options.